README.md

```diff
@@ -1,101 +1,5 @@
-#Topmodel BMI
-Developing a BMI topmodel function based on tmod9502.c, a C code written based on the original Beven Keith code Fortran code. See specifications for the original code at: https://csdms.colorado.edu/wiki/Model:TOPMODEL
+# TOPMODEL BMI
 
-tmod9502.c
-
-  TOPMODEL DEMONSTRATION PROGRAM VERSION 95.02
-
-  This C version by Fred Ogden, Sept. 2009.  Converted to ANSI C
-  and given somewhat more meaningful variable names.  Compared
-  against original FORTRAN version on a simple data set with no
-  infiltration excess.
-  
-  after Keith Beven 1985
-  Revised for distribution 1993,1995
-
-****************************************************************
-  This program is distributed freely with only two conditions.
```

Comment:
I presume these conditions for use are no longer applicable?

Reply:
No longer in README but original comment block still in [topmodel.c](https://github.com/NOAA-OWP/topmodel/blob/master/src/topmodel.c#L12).  Best to leave in source FORTRAN and C code but I'll ask about this.